go.mod

```diff
@@ -145,4 +145,6 @@ replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.5
 
 replace github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.15
 
+replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v0.0.0-20211202124722-2c356d6d98e4
```

Comment:
https://github.com/cosmos/cosmos-sdk/blob/master/db/go.mod#L36
cosmos-sdk use `v1.1.1`, should we follow suite here?